libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/PhotonUtils.java

```diff
@@ -58,7 +58,7 @@ public static String getPhotonImageUrl(String imageUrl, int width, int height, Q
         }
 
         // strip=all removes EXIF and other non-visual data from JPEGs
-        String query = "?strip=all";
+        String query = "?strip=info";
```

Comment:
The comment should be changed to "strip=info removes Exif, IPTC and comment data from the output image."

Reply:
Thanks for being on it, Nick! 🙇 addressed in 6db7cf4